fido-mds/src/mds.rs

```diff
@@ -488,6 +499,18 @@ pub enum AuthenticatorTransport {
     Internal,
 }
 
+impl fmt::Display for AuthenticatorTransport {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            AuthenticatorTransport::Usb => write!(f, "usb"),
```

Comment:
Are these intended to be human-readable values, like `AuthenticatorVersion`?

If so, they should be capitalized appropriately, and `FromStr for AuthenticatorTransport` should do case-insensitive matching.

Reply:
I think having them match the "standard" definition of the transport is better.